modules/tensor_mechanics/doc/content/modules/tensor_mechanics/C0TimoshenkoBeam.md

```diff
@@ -144,7 +144,7 @@ The axial and shear strain increments, and the rotational strain increments (aft
 \Delta M_2(x) = E \; \kappa_2(x)
 \end{equation}
 \begin{equation}
-\Delta M_3(x) = G \; \kappa_3(x)
+\Delta M_3(x) = E \; \kappa_3(x)
```

Comment:
@acasagran are you more familiar with the Timoshenko beam theory?

Reply:
@yw5aj, thanks for correcting this typo.

Reply:
Thank you @sveerara for reviewing!